Mobile version

ommarz opened the pull request:
[Add] Works and About Me sections
[Add] images for icons and backgrounds
[Positioning] flexbox and grid

ommarz committed:
[Add] [works and about me section]

Feature content

ommarz committed:
[Add] [images for background and content]

[Add] [images/content/ directory]

ommarz committed:
[Add] [About me section background image]

Imrovement in styling

ommarz committed:
[Add] [list icons]

AbayomiOlaoye commented on index.html:
```diff
@@ -46,4 +46,113 @@ <h3 class="presentation">I'm a Software Developer</h3>
+  </div>
+  </section>
+  <section class="about_me">
+    <div class="about_me_intro">
+      <h2 class="about_me_title">About me</h2>
+      <p class="about_me_description">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+      <a href="#" class="about_me_button">Get my resume</a>
+    </div>
+    <hr class="about_me_break">
+    <ul class="about_me_tools_list">
+      <li class="about_me_tool Language">Language</li>
+        <ul class="unit_tool_list">
+          <li class="language">JavaScript</li>
+          <li class="language">Ruby</li>
+          <li class="language">HTML</li>
+          <li class="language">CSS</li>
+        </ul>
+      <li class="about_me_tool Frameworks">Frameworks</li>
+        <ul class="unit_tool_list">
+          <li class="framework">React.js</li>
+          <li class="framework">Ruby on Rails</li>
+          <li class="framework">RSpec</li>
+          <li class="framework">Capybara</li>
+        </ul>
+      <li class="about_me_tool Skills">Skills</li>
+        <ul class="unit_tool_list">
+          <li class="skill">Database Management</li>
+          <li class="skill">Version Control</li>
+          <li class="skill">CLI</li>
+          <li class="skill">API Design</li>
+          <li class="skill">Web Design</li>
+        </ul>  
+    </ul>
+
```
|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/251997323-883ebf22-990a-4abd-98c0-765962638103.png)|![image](https://user-images.githubusercontent.com/99715640/251997433-3e7a92ca-8d65-4d76-b419-e339871fea32.png)|

- [x] Kindly ensure that you stick to the Figma design as much as you can in implementing the spacing/margin/padding issues earmarked in the figure above.

ommarz committed:
[REMOVE] [live demo from table of contents]

AbayomiOlaoye commented on index.html:
```diff
@@ -46,4 +46,113 @@ <h3 class="presentation">I'm a Software Developer</h3>
             class="social-icon"></img></a></li>
     </ul>
   </section>
+  <section class="works">
+    <h2 class="recent_works">My Recent Works</h2>
+    <hr class="recent_works_break">
+    <div class="main_project">
+      <img class="project_image" src="./images/content/project_main.svg" alt="project image">
+      <h3 class="work_title_main">Multi-Post Stories</h3>
+      <p class="work_description_main">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+      <ul class="tools_list_main">
+        <li class="tool_main">CSS</li>
+        <li class="tool_main">HTML</li>
+        <li class="tool_main">Bootstrap</li>
+        <li class="tool_main">Ruby</li>
+      </ul>
+      <a href="#" class="see_project_main">See Project</a>
+    </div>
+    <div class="project_card card1">
+        <h3 class="card_title">Professional Art<br> Printing Data</h3>
+        <p class="card_description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <ul class="tools_list_card">
+          <li class="tool_card">HTML</li>
+          <li class="tool_card">Bootstrap</li>
+          <li class="tool_card">Ruby</li>
+        </ul>
+        <a href="#" class="see_project_card">See Project</a>
+    </div>
```
- You have done really tremendous work with this project :clap: :clap: :muscle: 

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/252012601-c5f1b5a7-a34a-4d52-84d7-461f47c82958.png)|![image](https://user-images.githubusercontent.com/99715640/252011801-c112a726-6d7a-43a2-a460-a1c27dc5a71d.png)|

 - [x] How about reducing the space between the first and the second project cards to match Figma's design?
 - [x] Also, I think we are missing some color attributes in the first card. The project title and the paragraph is not as we have in the Figma design.
 - [x] Lastly, please note the overflow on the `y-axis`. It's not good for user experience

AbayomiOlaoye reviewed (changes requested):
## STATUS: CHANGE REQUIRED ♻️ 

Hi @ommarz :wave: :wave:  

Good job with this milestone so far! What a brilliant attempt!!
There are just a few more issues to work on before moving on to the next project but you are almost there!

![so_close](https://camo.githubusercontent.com/0c690ab997907a8c79afe7e695c891f6398cdbef93eaf8ce19f652ff321b54f8/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f744b797263304e546c4563475764546870342f67697068792e676966)

## Highlights
- Descriptive Pull Request message :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- Clean code, nice UI styles :heavy_check_mark: 
- Meaningful commit messages :heavy_check_mark: 
- No linter error :heavy_check_mark: 
- Descriptive ReadMe documentation :heavy_check_mark: 

## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ommarz committed:
[Fix] [styling in stylesheet.css]

Improvements to stick to FIGMA design
[Fix] background and text colors
[Fix] padding between <li> elements and bullets
[Fix] spacing between first and second card

ommarz committed:
[Edit] [typo]

ommarz committed:
[Fix] [overflow]

KanzaTahreem reviewed (approved):
# Approved 🙌

Hi @ommarz 

Good job implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**

<img src="https://media.tenor.com/xDHCe07zrocAAAAC/congrats-minions.gif"/>

## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great, perfect!
-  ✅ Great work using the right GitHub Flow 
-  ✅ Your README looks good, well done!
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
